packages/hebao_v1/contracts/modules/core/ForwarderModule.sol

```diff
@@ -65,17 +67,21 @@ contract ForwarderModule is BaseModule
         public
         view
     {
+        // Since this contract is a module, we need to prevent wallet from interacting with
+        // Stores via this module. Therefore, we must carefully check the 'to' address as follows,
+        // so no Store can be used as 'to'.
         require(
-            (to == from) ||
-            (to == controller.walletFactory()) ||
-            (to != address(this) && Wallet(from).hasModule(to)),
+            to == controller.walletFactory() || // 'from' can be the wallet to create (not its owner),
```

Comment:
> to == controller.walletFactory() || // 'from' can be the wallet to create (not its owner),

I guess this won't be possible because `verifySignature` is called on the wallet before the wallet is created in the meta tx call.

Reply:
Yes, I got the INVALID_SIGNATURE error now